templates/al2023_user_data.tpl

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: node.eks.aws/v1alpha1
+kind: NodeConfig
+spec:
+  cluster:
+    name: ${cluster_name}
+    apiServerEndpoint: ${cluster_endpoint}
+    certificateAuthority: ${cluster_auth_base64}
+    cidr: ${cluster_service_cidr}
+  kubelet:
+    flags:
+      - --node-labels=${node_labels}
+      - --register-with-taints=${node_taints}
+%{ if discard_unpacked_layers == false }
+  containerd:
+    config: |
+      [plugins."io.containerd.grpc.v1.cri".containerd]
+      discard_unpacked_layers = false
+%{ endif }
```

Comment:
This should probably be per asg instead of globally, or we'll have the same issues with VDCM as on our new environments.